Modules/Editors/ProjectCenter/PCEditor.m

```diff
@@ -1013,9 +1012,20 @@ - (void)editorTextViewWillPressKey:sender
 
 - (void)editorTextViewDidPressKey:sender
 {
-//  NSLog(@"Did pressing key");
   if (sender == _intEditorView || sender == _extEditorView)
-    [self computeNewParenthesisNesting: sender];
+    {
+      if (nil != phlTimer)
+	{
+	  [phlTimer invalidate];
+	  phlTimer = nil;
+	}
+
+      phlTimer = [NSTimer scheduledTimerWithTimeInterval:0.1
```

Comment:
@fredkiefer introduction of this timer (instead of delay) perfectly works around the problem we are seeing that the parenthesis closest to the last change does not change attributes.
I think this proves a problem with GUI (NSLayoutManager) since the attributes are calculated and set correctly.

This timer also allows the code ti invalidate and cope with quick typing and do less calculation, hopefully increasing speed, so I think it is worthwhile to keep. What do you think?

Reply:
Looks good to me.